src/picongpu/include/particles/ionization/ionizationMethods.hpp

```diff
@@ -63,7 +63,7 @@ namespace ionization
 
             /* each thread sets the multiMask hard on "particle" (=1) */
             childElectron[multiMask_] = 1;
-            const uint32_t weighting = parentIon[weighting_];
```

Comment:
but since all masses are scaled equally, and both are weighted with the same `weighting` this could still be skipped if one wants to spare the read. just for future notice.